addons/mail/views/assets.xml

```diff
@@ -110,6 +110,8 @@
                 <script type="text/javascript" src="/mail/static/src/components/thread_textual_typing_status/thread_textual_typing_status.js"></script>
                 <script type="text/javascript" src="/mail/static/src/components/thread_typing_icon/thread_typing_icon.js"></script>
                 <script type="text/javascript" src="/mail/static/src/components/thread_viewer/thread_viewer.js"></script>
+                <script type="text/javascript" src="/mail/static/src/components/composer_suggested_partners_list/composer_suggested_partners_list.js"></script>
+                <script type="text/javascript" src="/mail/static/src/components/composer_suggested_partner/composer_suggested_partner.js"></script>
```

Comment:
alphabetical order: swap because `composer_suggested_recipient` < `composer_suggested_recipient_list` (new names if you follow my requested changes)